Common/ImageSamplers/itkImageRandomSamplerSparseMask.hxx

```diff
@@ -38,7 +38,9 @@ ImageRandomSamplerSparseMask<TInputImage>::GenerateData()
   /** Sanity check. */
   if (mask == nullptr)
   {
-    itkExceptionMacro("ERROR: do not call this function when no mask is supplied.");
+    itkExceptionMacro("ERROR: do not call this function when no mask is supplied. When using the "
+                      "ImageRandomSamplerSparseMask sampler, a mask is required. Otherwise you may consider using a "
+                      "sampler that does not require a mask, for example, ImageRandomSampler.");
```

Comment:
@mstaring Can you please check if you like the proposed extended error message for `mask == nullptr`? Proposed text:

> ERROR: do not call this function when no mask is supplied. When using the ImageRandomSamplerSparseMask sampler, a mask is required. Otherwise you may consider using a sampler that does not require a mask, for example, ImageRandomSampler.

Reply:
prima